Altis_Life.Altis/core/admin/fn_adminFreeze.sqf

```diff
@@ -8,7 +8,7 @@
 if (FETCH_CONST(life_adminlevel) < 4) exitWith {closeDialog 0; hint localize "STR_ANOTF_ErrorLevel";};
 
 private _unit = lbData[2902,lbCurSel (2902)];
-_unit = call compile format ["%1", _unit];
+_unit = parseSimpleArray format ["%1", _unit];
```

Comment:
Again here

Reply:
don't touch these, I want to do them in a seperate pr (already prepared that one)